typescript/infra/config/environments/mainnet3/warp/configGetters/getEclipseEthereumApxETHWarpConfig.ts

```diff
@@ -0,0 +1,45 @@
+import { ethers } from 'ethers';
+
+import {
+  ChainMap,
+  OwnableConfig,
+  TokenRouterConfig,
+  TokenType,
+} from '@hyperlane-xyz/sdk';
+
+import { getOwnerConfigForAddress } from '../../../../../src/config/environment.js';
+import {
+  RouterConfigWithoutOwner,
+  tokens,
+} from '../../../../../src/config/warp.js';
+import { DEPLOYER } from '../../owners.js';
+
+const ethereumOwner = DEPLOYER;
+const eclipseOwner = '9bRSUPjfS3xS6n5EfkJzHFTRDa4AHLda8BU2pP4HoWnf';
+
+export async function getEclipseEthereumApxEthWarpConfig(
+  routerConfig: ChainMap<RouterConfigWithoutOwner>,
+  _abacusWorksEnvOwnerConfig: ChainMap<OwnableConfig>,
+): Promise<ChainMap<TokenRouterConfig>> {
+  const eclipsemainnet: TokenRouterConfig = {
+    ...routerConfig.eclipsemainnet,
+    ...getOwnerConfigForAddress(eclipseOwner),
+    type: TokenType.synthetic,
+    foreignDeployment: '9pEgj7m2VkwLtJHPtTw5d8vbB7kfjzcXXCRgdwruW7C2',
+    gas: 300_000,
```

Comment:
where does this number come from

Reply:
definitely is a magic number atm, will put it as a constant for now, but we should figure out a better way here